Library/Formula/znapzend.rb

```diff
@@ -0,0 +1,18 @@
+class Znapzend < Formula
+  desc "zfs backup with remote capabilities and mbuffer integration"
+  homepage "http://www.znapzend.org"
+  url "https://github.com/oetiker/znapzend/releases/download/v0.15.3/znapzend-0.15.3.tar.gz"
+  sha256 "27f7c218f4e71beac7b2ba52386a7742506d45c0b3fd40104f44ccef0f373b6e"
+
+  depends_on "perl" if MacOS.version <= :mavericks
+
+  def install
+    system "./configure", "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/znapzend", "-h"
```

Comment:
well, I thought about it. But I can't really now what the user's zfs datasets are, so I can't even try a dry-run, and I don't even know what to send where. I am out of ideas for a useful test because this thing needs to be properly setup. @ilovezfs Do you have ideas for a test?